public/apps/configuration/constants.tsx

```diff
@@ -240,6 +240,7 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/search/pipeline/delete',
   'cluster:admin/search/pipeline/get',
   'cluster:admin/search/pipeline/put',
+  'cluster:admin/security/resource/share',
```

Comment:
I don't think we should add this here. This only makes sense in the context of resource authz and not evaluated as a cluster perm like the rest of these.